File: app/src/main/java/com/example/sig/Objects/Note.java
package com.example.sig.Objects;

public class Note {
    private int id_note;
    private int id_user;
    private int id_object;
    private String description;
    private double note;

    public Note() {
    }

    public int getId_note() {
        return id_note;
    }

    public void setId_note(int id_note) {
        this.id_note = id_note;
    }

    public int getId_user() {
        return id_user;
    }

    public void setId_user(int id_user) {
        this.id_user = id_user;
    }

    public int getId_object() {
        return id_object;
    }

    public void setId_object(int id_object) {
        this.id_object = id_object;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public double getNote() {
        return note;
    }

    public void setNote(double note) {
        this.note = note;
    }
}

File: application/modules/ws/models/Note.php
<?php

class Note {
    private $_id_note;
    private $_id_user;
    private $_id_object;
    private $_note;

    //CONSTRUCTEURS
    public function __construct()
    {
        $ctp = func_num_args();
        $args = func_get_args();
        switch($ctp)
        {
            case 3:
                $this->_id_note = 0;
                $this->_id_user = $args[0];
                $this->_id_object = $args[1];
                $this->_note = $args[2];
            break;
            case 1:
                $this->hydrate($args[0]);
            break;
            default:
                break;
        }
    }

    //HYDRATATION
    public function hydrate(array $data)
    {
        foreach ($data as $key=>$value){
            $method='set'.ucfirst($key);
            if(method_exists($this,$method)){
                $this->$method($value);
            }
        }
    }

    /**
     * @return mixed
     */
    public function getId_Note(){ return $this->_id_note; }

    /**
     * @param mixed $id_note
     */
    public function setId_Note($id_note){ $this->_id_note = $id_note; }

    /**
     * @return mixed
     */
    public function getId_User(){ return $this->_id_user; }

    /**
     * @param mixed $id_user
     */
    public function setId_User($id_user){ $this->_id_user = $id_user; }

    /**
     * @return mixed
     */
    public function getId_Object(){ return $this->_id_object; }

    /**
     * @param mixed $id_object
     */
    public function setId_Object($id_object){ $this->_id_object = $id_object; }

    /**
     * @return mixed
     */
    public function getNote(){ return $this->_note; }

    /**
     * @param mixed $note
     */
    public function setNote($note){ $this->_note = $note; }
}